components/dash-table/package.json

```diff
@@ -17,7 +17,7 @@
     "private::build:js": "run-s \"private::build -- --mode production\"",
     "private::build:js-test": "run-s \"private::build -- --mode development --config webpack.test.config.js\"",
     "private::build:js-test-watch": "run-s \"private::build -- --mode development --config webpack.test.config.js --watch\"",
-    "private::build:backends": "dash-generate-components src/dash-table/dash/DataTable.js dash_table -p package-info.json && cp dash_table_base/** dash_table/ && dash-generate-components src/dash-table/dash/DataTable.js dash_table -p package-info.json --r-prefix 'dash' --r-suggests 'dash' --jl-prefix 'dash' && black dash_table",
+    "private::build:backends": "dash-generate-components src/dash-table/dash/DataTable.js dash_table -p package-info.json && cp dash_table_base/** dash_table/ && dash-generate-components src/dash-table/dash/DataTable.js dash_table -p package-info.json -k DataTable --r-prefix 'dash' --r-suggests 'dash' --jl-prefix 'dash' && black dash_table",
```

Comment:
If we make prop order explicit for table, I think we should be a little more careful about what order we end up with, especially since this will be reflected in the docs. The first two are right: `data` and `columns`. And it makes sense to keep `derived_*` and the dash standards (`loading_state`, `persistence`) at the end. Other than that it's currently a bit random, will be a bit hard for users to find what they want. @chriddyp do you want to give this a shot?